flutter/example/ios/Runner/Base.lproj/Main.storyboard

```diff
@@ -1,8 +1,10 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="10117" systemVersion="15F34" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" initialViewController="BYZ-38-t0r">
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@philipphofmann is this normal that things change on iOS project files, right? just want to be sure that I won't break anything :)

Reply:
I'm not an yet expert on this topic, maybe @HazAT can help us out here?